ftrr/src/main/java/framgia/co/edu/ftrr/entity/TitleInterview.java

```diff
@@ -1,72 +1,23 @@
 package framgia.co.edu.ftrr.entity;
-// Generated Feb 12, 2019 2:57:27 PM by Hibernate Tools 5.1.10.Final
+
+import lombok.Data;
 
 import javax.persistence.*;
 import java.util.Date;
 import java.util.List;
 
-/**
- * TitleInterviews generated by hbm2java
- */
+@Data
 @Entity
 @Table(name = "title_interviews", catalog = "FTRR")
 public class TitleInterview implements java.io.Serializable {
-
-	private int id;
-	private String name;
-	private Date deletedAt;
-	private List<ResultInterview> resultInterviewses;
-
-	public TitleInterview() {
-	}
-
-	public TitleInterview(int id) {
-		this.id = id;
-	}
-
-	public TitleInterview(int id, String name, Date deletedAt, List<ResultInterview> resultInterviewses) {
-		this.id = id;
-		this.name = name;
-		this.deletedAt = deletedAt;
-		this.resultInterviewses = resultInterviewses;
-	}
-
-	@Id
-	@Column(name = "id", unique = true, nullable = false)
-	public int getId() {
-		return this.id;
-	}
-
-	public void setId(int id) {
-		this.id = id;
-	}
-
-	@Column(name = "name")
-	public String getName() {
-		return this.name;
-	}
-
-	public void setName(String name) {
-		this.name = name;
-	}
-
-	@Temporal(TemporalType.TIMESTAMP)
-	@Column(name = "deleted_at", length = 19)
-	public Date getDeletedAt() {
-		return this.deletedAt;
-	}
-
-	public void setDeletedAt(Date deletedAt) {
-		this.deletedAt = deletedAt;
-	}
-
-	@OneToMany(fetch = FetchType.LAZY, mappedBy = "titleInterviews")
-	public List<ResultInterview> getResultInterviewses() {
-		return this.resultInterviewses;
-	}
-
-	public void setResultInterviewses(List<ResultInterview> resultInterviewses) {
-		this.resultInterviewses = resultInterviewses;
-	}
-
+    @Id
+    @Column(name = "id", unique = true, nullable = false)
+    private int id;
+    @Column(name = "name")
+    private String name;
+    @Temporal(TemporalType.TIMESTAMP)
+    @Column(name = "deleted_at", length = 19)
+    private Date deletedAt;
```

Comment:
chỗ này chị nghĩ là ` Timestamp` chứ em hì, entity thì lưu Date nó đâu đủ he

nếu đúng thì check hết toàn bộ field: created_at, updated_at, deleted_at của entity nè :p